githubbot/main.go

```diff
@@ -22,6 +22,8 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+const botName = "githubbot"
```

Comment:
can we not hardcode this and just stick `s.kbc.GetUsername()` somewhere in the opts since we already run that?